source/user-manual/api/getting-started.rst

```diff
@@ -898,6 +898,36 @@ Adding an agent is now easier than ever. Simply send a request with the agent na
       "error": 0
     }
 
+Ingest security events
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.6.0
+
+You can send security events for analysis using the Wazuh API.
+
+.. code-block:: console
+
+    # curl -k -X POST "https://localhost:55000/events" -H  "Authorization: Bearer $TOKEN" -H  "Content-Type: application/json" -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'
+
+.. code-block:: json
+    :class: output
+
+    {
+      "data": {
+        "affected_items": [
+
+        ],
+        "total_affected_items": 2,
+        "total_failed_items": 0,
+        "failed_items": []
+      },
+      "message": "All events were forwarded to analisysd",
+      "error": 0
+    }
+
+.. note::
+
+   You can send a maximum of 100 events per request.
```

Comment:
I just noticed that we are not documenting the 30 requests/minute rate limit for this endpoint. 

Maybe we should include in this note a notice that the endpoint is not meant to ingest large amounts of logs and is therefore limited to 30 requests per minute and 100 events per request. I would also specify the 30 requests/minute on the configuration page (`max_request_per_minute`):
- https://documentation.wazuh.com/current/user-manual/api/configuration.html#access